### DISCUSSION TITLE
FakeDNS、tproxy 透明代理、UDP ， 这三个条件筹到一起，Xray 的处理是不是有问题。

### DISCUSSION THREAD

**Discussion by wy16W2pIilK1xgqN** [XTLS/Xray-core#1786] [Q&A]
访问流程：软件 使用 UDP 访问 FakeDNS给的网络地址，被路由器 tproxy 劫持到 Xray  处理。

使用 Chrome 打开谷歌系的网站，经常摸莫名其妙的 访问不了，同一时间 Firefox 就正常。 
如果这时改用  Chrome 使用 SOCKS5  代理，也没有问题。使用同样内核  Edge、vivaldi 也有相同的问题。
随着 QUIC 和HTTP3 普及，这个问题愈发明显，现在只能关闭 UDP 代理使用了。 

 V2ray 发布  FakeDNS 功能时，我就这样用了，一直有这个问题，也发现不是我的个例。
这个月换到 Xray ，还是有一样的问题。
大家有没有根治的办法。

**Comment by chika0801** [comment]
你不用quic(http3)吧，因为现在各翻墙协议对它加速效果都没很好的解决方法。
在路由器上的插件如passwall默认是屏蔽了udp443端口。
